.github/scripts/init-pytest-ui.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "UI pre-run: ensuring pytest accepts charm-file argument"
+
+# Create a temporary conftest.py that adds the --charm-file flag
+cat > tests/conftest.py <<'EOF'
+def pytest_addoption(parser):
+    parser.addoption("--charm-file", action="store", default=None)
+EOF
```

Comment:
We can actually have a `conftest.py` in the `tests/` dir, similar to [this](https://github.com/canonical/temporal-k8s-operator/blob/track/1.23/tests/conftest.py).